decorum/decorum.py

```diff
@@ -63,6 +64,10 @@ def init(self, *args, **kwargs):
 
 
 def decorator(cls):
+    warnings.warn("decorum.decorator() will be deprecated. "
```

Comment:
I'm okay with promoting the class-based approach, but I'd prefer to continue to support this (as it's actually my preferred way to create decorators...I enjoy that I get to use a decorator to do so).